test/mongoose_push_application_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule MongoosePushApplicationTest do
+  use ExUnit.Case
+  import MongoosePush.Application
+  doctest MongoosePush.Application
+
+  test "worker name" do
+    assert :apns_name1_1 == worker_name(:apns, :name1, 1)
+    assert :fcm_name2_12 == worker_name(:fcm, :name2, 12)
+  end
+
+  test "pool groups" do
+    assert [:dev1, :dev2] == pools_by_mode(:apns, :dev)
+    assert [:prod1, :prod2] == pools_by_mode(:apns, :prod)
```

Comment:
I've put some assertions in setup, but I don't check every single thing in the config. There are only some general pools checks to show that they are needed.